PIL/JpegImagePlugin.py

```diff
@@ -593,7 +593,7 @@ def _save(im, fp, filename):
 
     info = im.encoderinfo
 
-    dpi = info.get("dpi", (0, 0))
+    dpi = [int(x) for x in info.get("dpi", (0, 0))]
```

Comment:
Is this essentially what you're referring to? https://github.com/python-pillow/Pillow/blob/master/PIL/BmpImagePlugin.py#L104